### PR TITLE
thumbs bugfix: audthumbs toggle, presenter success seen

### PR DESCRIPTION
**Pull request by AriLFrankel**

### CODE REVIEW

File: client/src/components/AudThumbs.jsx [hunk 1]
```diff
@@ -24,16 +24,11 @@ class AudThumbs extends Component {
       socket.emit('thumb clicked', topicId, userId, thumbChoice);
       // fade out component and set 'displayed' property to false in the store
       $('#Thumbs').fadeToggle('slow');
+      // dispatch
```
Comment (rtopol2) [inline client/src/components/AudThumbs.jsx]: this comment needs finishing

File: client/src/components/AudienceView.jsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ import AudThumbs from './AudThumbs';
 class AudienceView extends Component {
 
   componentDidMount () {
+    let thumbsDisplayed = false;
     let thumbs = this.props.thumbs;
```
Comment (rtopol2) [inline client/src/components/AudienceView.jsx]: should take out this line since it's not being used anymore

**Comment by JTMcDuff** [comment]
Pending Ross' approval, this looks good to go.

**Comment by JTMcDuff** [comment]
Gonna go ahead and merge.

**Pull request merged**